nilearn/decomposition/canica.py

```diff
@@ -106,30 +109,31 @@ class CanICA(MultiPCA, CacheMixin):
       datasets", IEEE ISBI 2010, p. 1177
     """
 
-    def __init__(self, mask=None, n_components=20,
-                 smoothing_fwhm=6, do_cca=True,
-                 threshold='auto', n_init=10,
-                 standardize=True,
-                 random_state=0,
-                 target_affine=None, target_shape=None,
+    def __init__(self, n_init=1, threshold='auto', n_components=20,
```

Comment:
I reordered the parameters so that they are consistent with MultiNiftiMasker parameters (an instance of MultiNiftiMasker being created)